.changeset/olive-otters-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/app-bridge": patch
+---
+
+feat: useTemplateAssets hook to return themeAsset as well
```

Comment:
Please add all the changes to `main`, only breaking changes on AppBridgeBlock should land in `dev` as it is used by the public.
Then you can proceed to merge `main` into `dev` and publish the v4 to avoid type issues

Reply:
The change is already in main #834.
After the release of v3 i created this PR.

Reply:
Ah ok, the title and description made me think it was new code